src/Microsoft.SqlTools.Hosting/Hosting/Protocol/Constants.cs

```diff
@@ -16,6 +16,9 @@ public static class Constants
         public static readonly string DefaultApplicationName = "azdata";
         public static readonly string SqlLoginAuthenticationType = "SqlLogin";
 
+        // Feature names
+        public static readonly string languageServiceFeature = "languageService";
```

Comment:
nit: following naming convention of other public constants this should be `LanguageServiceFeature`.